lib/vsc/install/shared_setup.py

```diff
@@ -1357,6 +1357,7 @@ def action_target(target, setupfn=setup, extra_sdist=[], urltemplate=None):
             'setuptools'
         ],
         'excluded_pkgs_rpm': [], # vsc-install ships vsc package (the vsc package is removed by default)
+        'namespace_packages': ['vsc'],
```

Comment:
my understanding of a namespace package is that the namespace itself is empty. so as soon as we remove `vsc.fancylogger`, this is ok.
@JensTimmerman is this a good time to remove `vsc.fancylogger`? we should be able to check logstash for any leftover usage.